src/phases/select-modifier-phase.ts

```diff
@@ -9,16 +9,20 @@ import i18next from "i18next";
 import * as Utils from "#app/utils";
 import { BattlePhase } from "./battle-phase";
 import Overrides from "#app/overrides";
+import { CustomModifierSettings } from "#app/modifier/modifier-type";
+import { isNullOrUndefined } from "#app/utils";
 
 export class SelectModifierPhase extends BattlePhase {
   private rerollCount: integer;
   private modifierTiers: ModifierTier[];
+  private customModifierSettings?: CustomModifierSettings;
 
-  constructor(scene: BattleScene, rerollCount: integer = 0, modifierTiers?: ModifierTier[]) {
+  constructor(scene: BattleScene, rerollCount: integer = 0, modifierTiers?: ModifierTier[], customModifierSettings?: CustomModifierSettings) {
     super(scene);
 
     this.rerollCount = rerollCount;
     this.modifierTiers = modifierTiers!; // TODO: is this bang correct?
```

Comment:
```suggestion
    this.modifierTiers = modifierTiers;
```
Smh flx not even trying to fix the simple `!` when he made the `strict-null` PR.